lib/dealership.rb

```diff
@@ -0,0 +1,20 @@
+class Dealership
+  attr_reader :name, :address, :inventory
+
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+  end
+
+  def inventory_count
+    inventory = 0
+  end
```

Comment:
Since we are storing all of our inventory in an array we can get the count of the inventory by calling the `.count` (or `.size` or `length they all work the same) method on the array. 

```suggestion
  def inventory_count
    @inventory.count
  end
```